src/dfcx_scrapi/builders/agents.py

```diff
@@ -54,52 +61,14 @@ def __str__(self) -> str:
             f"\nsecurity_settings: {self.proto_obj.security_settings}")
 
 
-    def _check_agent_exist(self):
-        """Check if the proto_obj exists otherwise raise an error."""
-        if not self.proto_obj:
-            raise ValueError(
-                "There is no proto_obj!"
-                "\nUse create_new_agent or load_agent to continue."
-            )
-        elif not isinstance(self.proto_obj, Agent):
-            raise ValueError(
-                "proto_obj is not an Agent type."
-                "\nPlease create or load the correct type to continue."
-            )
-
-
-    def load_agent(
```

Comment:
If we're removing `load_agent()` are we replacing it with anything? If not, are we losing anything by removing it?